docs/faststore/docs/sdk/analytics/use-analytics-event.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: "useAnalyticsEvent"
+---
+
+The `useAnalyticsEvent` hook intercepts both native and custom events triggered by the [`sendAnalyticsEvent`](https://developers.vtex.com/docs/guides/faststore/analytics-send-analytics-event) function. The hook automatically detects the events sent by `sendAnalyticsEvent`) and provides a response to them.
```

Comment:
question: I don't know if the ")" was mistakenly added or if you were going to add a link or something 😄 
```suggestion
The `useAnalyticsEvent` hook intercepts both native and custom events triggered by the [`sendAnalyticsEvent`](https://developers.vtex.com/docs/guides/faststore/analytics-send-analytics-event) function. The hook automatically detects the events sent by `sendAnalyticsEvent` and provides a response to them.
```

Reply:
It was a msitake @lariciamota 😅 Thanks for noticing it